pkg/translator/loki/convert.go

```diff
@@ -120,6 +126,29 @@ func convertLogToJSONEntry(lr plog.LogRecord, res pcommon.Resource) (*logproto.E
 	}, nil
 }
 
+func convertLogToLogfmtEntry(lr plog.LogRecord, res pcommon.Resource) (*logproto.Entry, error) {
+	line, err := EncodeLogfmt(lr, res)
+	if err != nil {
+		return nil, err
+	}
+	return &logproto.Entry{
+		Timestamp: timestampFromLogRecord(lr),
+		Line:      line,
+	}, nil
+}
+
+func convertLogToLokiEntry(lr plog.LogRecord, res pcommon.Resource, format string) (*logproto.Entry, error) {
+	switch format {
+	case formatJson:
+		return convertLogToJSONEntry(lr, res)
+	case formatLogfmt:
+		return convertLogToLogfmtEntry(lr, res)
+	default:
```

Comment:
Instead of failing, perhaps we could just compare format with "logfmt" and use convertLogToLogfmtEntry in that case, using `convertLogToJSONEntry` for all other cases?

Reply:
Can do, but are you certain this is the best way to go? Unexpected value would certainly be a bug on caller/client side, most likely best fixed. Is there a concern of losing logs due to this error?